evap/rewards/tools.py

```diff
@@ -25,11 +26,19 @@
 
 @login_required
 @transaction.atomic
-def save_redemptions(request, redemptions: Dict[int, int]):
+def save_redemptions(request, redemptions: Dict[int, int], previous_reward_points):
```

Comment:
This should be `previous_redeemed_points`, or? And let's add a type annotation :)

Reply:
(this also closes the comment from @He3lixxx about the name)